HLTrigger/Egamma/plugins/HLTEgammaAllCombMassFilter.cc

```diff
@@ -56,58 +54,64 @@ bool HLTEgammaAllCombMassFilter::hltFilter(edm::Event& iEvent, const edm::EventS
+      if (mass >= minMass_)
+        accept = true;
     }
   }
-  for(size_t firstLegNr=0;firstLegNr<firstLegP4s.size();firstLegNr++){
-    for(size_t secondLegNr=0;secondLegNr<firstLegP4s.size();secondLegNr++){
+  for (size_t firstLegNr = 0; firstLegNr < firstLegP4s.size(); firstLegNr++) {
+    for (size_t secondLegNr = 0; secondLegNr < firstLegP4s.size(); secondLegNr++) {
       math::XYZTLorentzVector pairP4 = firstLegP4s[firstLegNr] + firstLegP4s[secondLegNr];
       double mass = pairP4.M();
-      if(mass>=minMass_) accept=true;
+      if (mass >= minMass_)
+        accept = true;
     }
   }
-  for(size_t firstLegNr=0;firstLegNr<secondLegP4s.size();firstLegNr++){
-    for(size_t secondLegNr=0;secondLegNr<secondLegP4s.size();secondLegNr++){
+  for (size_t firstLegNr = 0; firstLegNr < secondLegP4s.size(); firstLegNr++) {
+    for (size_t secondLegNr = 0; secondLegNr < secondLegP4s.size(); secondLegNr++) {
       math::XYZTLorentzVector pairP4 = secondLegP4s[firstLegNr] + secondLegP4s[secondLegNr];
       double mass = pairP4.M();
-      if(mass>=minMass_) accept=true;
+      if (mass >= minMass_)
+        accept = true;
     }
   }
 
   return accept;
 }
 
-void  HLTEgammaAllCombMassFilter::getP4OfLegCands(const edm::Event& iEvent, const edm::EDGetTokenT<trigger::TriggerFilterObjectWithRefs>& filterToken, std::vector<math::XYZTLorentzVector>& p4s)
-{
+void HLTEgammaAllCombMassFilter::getP4OfLegCands(
+    const edm::Event& iEvent,
+    const edm::EDGetTokenT<trigger::TriggerFilterObjectWithRefs>& filterToken,
+    std::vector<math::XYZTLorentzVector>& p4s) {
   edm::Handle<trigger::TriggerFilterObjectWithRefs> filterOutput;
-  iEvent.getByToken(filterToken,filterOutput);
+  iEvent.getByToken(filterToken, filterOutput);
 
   //its easier on the if statement flow if I try everything at once, shouldnt add to timing
   std::vector<edm::Ref<reco::RecoEcalCandidateCollection> > phoCands;
-  filterOutput->getObjects(trigger::TriggerPhoton,phoCands);
+  filterOutput->getObjects(trigger::TriggerPhoton, phoCands);
   std::vector<edm::Ref<reco::RecoEcalCandidateCollection> > clusCands;
-  filterOutput->getObjects(trigger::TriggerCluster,clusCands);
+  filterOutput->getObjects(trigger::TriggerCluster, clusCands);
   std::vector<edm::Ref<reco::ElectronCollection> > eleCands;
-  filterOutput->getObjects(trigger::TriggerElectron,eleCands);
+  filterOutput->getObjects(trigger::TriggerElectron, eleCands);
 
-  if(!phoCands.empty()){ //its photons
-    for(auto & phoCand : phoCands){
+  if (!phoCands.empty()) {  //its photons
+    for (auto& phoCand : phoCands) {
       p4s.push_back(phoCand->p4());
     }
-  }else if(!clusCands.empty()){ //try trigger cluster (should never be this, at the time of writing (17/1/11) this would indicate an error)
-    for(auto & clusCand : clusCands){
+  } else if (!clusCands
+                  .empty()) {  //try trigger cluster (should never be this, at the time of writing (17/1/11) this would indicate an error)
```

Comment:
@smuzaffar in principle als this case looks a bit bizarre, always because of the long comment